client/app/bundles/shared/components/Avatar.scss

```diff
@@ -0,0 +1,25 @@
+@import './base';
+
+.avatar {
+  display: inline-block;
+
+  font-family: $font-family;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+
+  img {
+    border-radius: 50%;
+    border: 5px solid;
+    border-color: $background-color;
+  }
```

Comment:
Let's add a width for these! Perhaps something like:

```css
img {
  /* ... */
  width: 100%;
  max-width: 120px;
}
```

Reply:
@nshki Done!